### ISSUE TITLE
Instrument MadMiner code with MLflow Tracking

### ISSUE THREAD

**Issue by BenGalewsky** [scailfin/madminer-workflow#23]
_An an analyzer I want to see the performance and generated artifacts from my MadMiner run so I can evaluate the fit and inspect the artifacts_

# Assumptions
1. MLflow Tracking logging statements added to the model code. These statements write loss functions and other measurements alongside the input hyperparameters.

# Acceptance Criteria
1. At the end of a run I can view the MLflow UI and inspect the results
2. At the end of a run I can view generated artifacts

**Comment by Sinclert** [comment]
The main PR has been merged.

Feel free to close this issue (I cannot).
